homework/src/index.js

```diff
@@ -30,15 +34,96 @@
     return elem;
   }
 
+  function addRow(label, value, tbody) {
+    const tr = createAndAppend('tr', tbody);
+    createAndAppend('td', tr, { class: 'label', text: label });
+    createAndAppend('td', tr, { text: value });
+  }
+
+
+  function renderRepository(info, container) {
+    container.innerHTML = '';
+    const table = createAndAppend('table', container);
+    const tbody = createAndAppend('tbody', table);
+
+    const tr1 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr1, {
+      text: 'Repository: ',
+    });
+    const repository = createAndAppend('td', tr1);
+    createAndAppend('a', repository, {
+      href: info.html_url,
+      target: '_blank',
+      text: info.name
+    });
+
+    const descriptions = `${info.description} `;
+    addRow('Description: ', descriptions, tbody);
+    const forks = `${info.forks} `;
+    addRow('Forks: ', forks, tbody);
+    const updated = `${info.updated_at} `;
+    addRow('Updated: ', updated, tbody);
+
+  }
+
+  function renderContributor(info, container) {
+    container.innerHTML = '';
+    const h3 = createAndAppend('h3', container, { text: 'Contributors' });
+    const ul = createAndAppend('ul', container);
+
+    info.forEach(contributor => {
+      const li1 = createAndAppend('li', ul);
+      createAndAppend('img', li1, { src: contributor.avatar_url });
+      const a = createAndAppend('a', li1, { href: contributor.html_url, target: '_blank' });
+      a.innerText = `${contributor.login}`;
+      const li2 = createAndAppend('li', ul);
+      li2.innerText = `${contributor.contributions}`;
+    });
+  }
+
   function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
+    fetchJSON(url)
+      .then(data => {
```

Comment:
`data` is a meaningless name as I have mentioned several times in class. In this case, the best name is `repositories`.